docs/src/main/asciidoc/health-guide.adoc

```diff
@@ -56,141 +58,171 @@ which is an implementation of the MicroProfile Health specification used in Quar
+Importing the `smallrye-health` extension directly exposes three REST endpoints:
+
+- `/health/live` - The application is up and running.
+- `/health/ready` - The application is ready to serve requests.
+- `/health` - Accumulating all health check procedures in the application.
+ 
+To check that smallrye-health extension is working as expected:
 
 * start your Quarkus application with `mvn compile quarkus:dev`
-* access the `http://localhost:8080/health` endpoint using your browser or 
-`curl http://localhost:8080/health`
+* access the `http://localhost:8080/health/live` endpoint using your browser or 
+`curl http://localhost:8080/health/live`
 
-The health REST enpoint returns a simple JSON object with two fields:
+All of the health REST endpoints return a simple JSON object with two fields:
 
 * `status` -- the overall result of all the health check procedures
 * `checks` -- an array of individual checks
 
-The general `status` of the health check is computed as a logical AND of all the declared
-health check procedures. The `checks` array is empty as we have not specified any health 
-check procedure yet so let's define some.
+The general `status` of the health check is computed as a logical AND of all the 
+declared health check procedures. The `checks` array is empty as we have not specified 
+any health check procedure yet so let's define some.
 
 == Creating your first health check
 
-In this section we create our first simple health check procedure.
+In this section, we create our first simple health check procedure.
 
 Create the `org.acme.health.SimpleHealthCheck` class:
 
 [source,java]
 ----
 package org.acme.health;
 
-import org.eclipse.microprofile.health.Health;
 import org.eclipse.microprofile.health.HealthCheck;
 import org.eclipse.microprofile.health.HealthCheckResponse;
+import org.eclipse.microprofile.health.Liveness;
 
 import javax.enterprise.context.ApplicationScoped;
 
-@Health
+@Liveness
 @ApplicationScoped
 public class SimpleHealthCheck implements HealthCheck {
-    
+
     @Override
     public HealthCheckResponse call() {
         return HealthCheckResponse.named("Simple health check").up().build();
     }
 }
 ----
 
-As you can see health check procedures are defined as implementations of the `HealthCheck` 
-interface which are defined as CDI beans with the `@Health` qualifier. `HealthCheck` is 
-a functional interface whose single method `call` returns a `HealthCheckResponse` object 
-which can be easily constructed by the fluent builder API shown in the example.
+As you can see health check procedures are defined as implementations of the 
+`HealthCheck` interface which are defined as CDI beans with the one of the 
+following CDI qualifiers: 
+
+- `@Liveness` - the liveness check accessible at `/health/live`
+- `@Readiness` - the readiness check accessible at `/health/ready`
+- `@Health` - *deprecated* unspecified health procedure accessible at `/health`
```

Comment:
I would remove this one: Quarkus is not 1.0 yet so let's not advertise deprecated features.